### ISSUE TITLE
Needs Update 

### ISSUE THREAD

**Issue by pinkparrot** [MicrosoftDocs/xamarin-docs#941]
Thank you for this amazing, well organized guide. I've learned lots of things but I think it's out of date now. Therefore, the given code above doesn't work properly. Please update the guide.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5b76624d-7d4a-93bb-a6e2-72ad62bd7d2c
* Version Independent ID: 86ba4c5c-b279-88f5-0c3b-a0ae010c45e8
* Content: [Hello, iOS Multiscreen – Quickstart - Xamarin](https://docs.microsoft.com/en-us/xamarin/ios/get-started/hello-ios-multiscreen/hello-ios-multiscreen-quickstart?pivots=macos)
* Content Source: [docs/ios/get-started/hello-ios-multiscreen/hello-ios-multiscreen-quickstart.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/ios/get-started/hello-ios-multiscreen/hello-ios-multiscreen-quickstart.md)
* Product: **xamarin**
* GitHub Login: @lobrien
* Microsoft Alias: **laobri**

**Comment by conceptdev** [comment]
Thanks for your feedback. This guide was last updated just a few weeks ago, including testing the sample. Can you let us know which part didn't work for you?

**Comment by conceptdev** [comment]
As there's been no activity on this issue, I'm going to close it. If you find some code in the document does not work properly, please add additional comments or open a new issue.
Thanks!

**Comment by yongauh** [comment]
In PrepareForSegue,

var callHistoryController = segue.DestinationViewController as CallHistoryController;

returns null. So I get an exception in Main.

**Comment by lobrien** [comment]
I compiled and ran https://github.com/xamarin/ios-samples/tree/master/Hello_iOS/Hello.iOS_MultiScreen without problem. I checked the source code (particularly the storyboard) and it looks correct. 

This solution depends upon the segue being defined in the storyboard @yongauh and needs to be run with the storyboard as defined in the sample directory. Perhaps you're trying to recreate the solution from scratch? That would lead to (at least) the error you report.
